doc/source/user_guide/timeseries.rst

```diff
@@ -2220,6 +2220,32 @@ you can use the ``tz_convert`` method.
 
    rng_pytz.tz_convert('US/Eastern')
 
+.. note::
```

Comment:
let's make this a sub-section (e.g. Fold), add a reference (so you can link to it in whatsnew)

Reply:
should be the same level as working with time zones

Reply:
Done.